Jenkinsfile

```diff
@@ -8,6 +8,19 @@ pipeline {
     }
 
     stages {
+        stage('Common Module Unit Tests') {
+            steps {
+                dir('common') {
+                    sh label: 'Installing dependencies', script: 'pipenv install --dev --deploy --ignore-pipfile'
+                    sh label: 'Running unit tests', script: 'pipenv run unittests-cov'
+                    sh label: 'Displaying code coverage report', script: 'pipenv run coverage-report'
+                    sh label: 'Exporting code coverage report', script: 'pipenv run coverage-report-xml'
+                    cobertura coberturaReportFile: '**/coverage.xml'
```

Comment:
Optional: seeing as this is the same as the MHS unit tests block, you possibly could [extract as a function](https://stackoverflow.com/a/47631522), though I'm not sure how well supported that functionality is.

Reply:
I've added this in since we'll probably have similar commands for the SCR adapter as well it makes sense to have it